eng/testing/workloads-browser.targets

```diff
@@ -70,7 +71,7 @@
       <_NuGetsToBuild Include="$(_DefaultRuntimePackNuGetPath)"
                       Project="$(InstallerProjectRoot)pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj"
                       Dependencies="$(_DefaultRuntimePackNuGetPath)"
-                      Properties="@(_DefaultPropsForNuGetBuild, ';');WasmEnableThreads=$(WasmEnableThreads)"
+                      Properties="@(_DefaultPropsForNuGetBuild, ';');WasmEnableThreads=$(_IsMTNugetMissing)"
```

Comment:
WasmEnableThreads=!WasmEnableThreads is a lie. Why do we need it ? Could we do better ?

Reply:
We could go back to the original version where we have 2 `_NuGetsToBuild` and a condition. One would be run for `WasmEnableThreads=true` and pass `WasmEnableThreads=false` to the `Properties`, the other would do the reverse. However, it's code duplication.

Reply:
@akoeplinger could you please help Ilona to figure this out ? 
It's some trick with creating fake nuget, but we don't understand the details.

I worry that this lie will trip-up somebody reading binlog and waste many hours trying to figure it out.

Reply:
The logic is correct and previously there was the similar thing - "build the other flavors"

Reply:
But the comment is already there

> For local builds, only one of the 3 required runtime packs might be available. In that case,
> build the other nugets with the *same runtime* but different names.

Maybe we can fix the "3", because there are only "2" currently

Reply:
Hehe, comment already fixed :)

Reply:
Comment would help a lot. Thanks

Reply:
1) Moving the comment closer to `_NuGetsToBuild`. We can keep the original one at the beginning of the Task.
2) The comment above `_NuGetsToBuild`:
"We should have both: multithreaded and singlethreaded nugets in local packages. The one that is in line with the runtime type is already there, here we create the missing one."

Reply:
"We need nugets for all wasm runtime flavors. The one corresponding the current property values is already built, the others needs to be add to the `_NuGetsToBuild`"?